src/main/java/org/folio/circulation/domain/Request.java

```diff
@@ -99,6 +100,15 @@ boolean isFulfillable() {
     return getFulfilmentPreference() == HOLD_SHELF || getFulfilmentPreference() == DELIVERY;
   }
 
+  public boolean isPage() {
+    return getRequestType() == RequestType.PAGE;
+  }
+
+  public boolean isTitleLevelRequest() {
+    return this.getRequestLevel() == TITLE &&
+      this.getTlrSettingsConfiguration() != null &&
+      this.getTlrSettingsConfiguration().isTitleLevelRequestsFeatureEnabled();
+  }
```

Comment:
what is the difference between this and isTitleLevel() ?

Reply:
Done